css-ecommerce-page/index.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Ecommerce Challenge</title>
+  <link href="https://fonts.googleapis.com/css?family=DM+Sans&display=swap" rel="stylesheet">
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="column-full">
+        <div>
+          <h1>New Arrivals</h1>
+        </div>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="column-fifth">
+        <div class="mid">
+          <img src="./images/dress1.jpeg" alt="dress1">
+        </div>
+      </div>
+
+      <div class="column-fifth">
+        <div class="mid">
+          <img src="./images/ysl-bag.jpeg" alt="ysl-bag">
+        </div>
+      </div>
+
+      <div class="column-fifth">
+        <div class="mid">
+          <img src="./images/key.jpeg" alt="key">
+        </div>
+      </div>
+
+      <div class="column-fifth">
+        <div class="mid">
+          <img src="./images/lv-bag.jpg" alt="lv-bag" id="contain">
+        </div>
+      </div>
+
+      <div class="column-fifth">
+        <div class="mid">
+          <img src="./images/dress2.jpeg" alt="dress2">
+        </div>
+      </div>
+    </div>
```

Comment:
I can understand why you would do this from a layout class perspective, but these images are related to the text below them, and dividing your page like this makes them seem unrelated when reading the HTML document. I suggest you keep the images and text together inside one `column-fifth`

Reply:
![image](https://user-images.githubusercontent.com/117887912/208336419-8cc56c9d-79dc-4461-a74f-945209c1dc35.png)